src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px 16px;
+
+  font-family: Roboto;
+  font-weight: 400;
+
+  &__image {
+    display: flex;
+    justify-self: center;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    // width: 160px;
+    height: 134px;
+    margin-inline: 3px;
+    margin-bottom: 40px;
+    padding: 0;
+    background-origin: border-box;
+  }
+
+  &__text {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+    color: $main_accent-color;
+  }
+
+  &__text--product-code {
+    color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used here. Make sure this variable is defined in your SCSS files to avoid compilation errors.